l2geth/miner/worker.go

```diff
@@ -781,6 +784,7 @@ func (w *worker) commitTransactions(txs *types.TransactionsByPriceAndNonce, coin
 	}
 
 	var coalescedLogs []*types.Log
+	var txCount int
```

Comment:
should add rationale for why this is needed, as well as updating the godoc to indicate that this method returns true on failure.